packages/module/patternfly-docs/content/extensions/component-groups/examples/ServiceCard/ServiceCard.md

```diff
@@ -0,0 +1,28 @@
+---
+section: extensions
+subsection: Component groups
+id: Service card
+source: react
+propComponents: ['ServiceCard']
+sourceLink: https://github.com/patternfly/react-component-groups/blob/main/packages/module/patternfly-docs/content/extensions/component-groups/examples/ServiceCard/ServiceCard.md
+---
+
+import ServiceCard from "@patternfly/react-component-groups/dist/dynamic/ServiceCard";
+import { EllipsisVIcon } from '@patternfly/react-icons';
+import contentHeaderIcon from '../../assets/icons/content-header-icon.svg'
+
+A **Service Card** component allows
+
+## Examples
+
+### Service Card
+
+```js file="./ServiceCardExample.tsx"
+
+```
+
+### Service Card with Gallery example
+
+```js file="./ServiceCardGalleryExample.tsx"
+
+```
```

Comment:
@InsaneZein can you please add some texts to the docs? It would also be nice to create an example with the custom footer. Let me know if you need help with the texts.